metadata-ingestion/src/datahub/ingestion/source/openapi_parser.py

```diff
@@ -357,12 +360,15 @@ def get_tok(
     url4req = url + tok_url
     if method == "post":
         # this will make a POST call with username and password
-        data = {"username": username, "password": password}
+        data = {"username": username, "password": password, "maxDuration": True}
         # url2post = url + "api/authenticate/"
-        response = requests.post(url4req, data=data)
+        response = requests.post(url4req, proxies=proxies, json=data)
         if response.status_code == 200:
             cont = json.loads(response.content)
-            token = cont["tokens"]["access"]
+            if "token" in cont:  # other authentication scheme
+                token = cont["token"]
+            else:  # works only for bearer authentication scheme
+                token = "Bearer" + cont["tokens"]["access"]
```

Comment:
looks like this is missing a space
```suggestion
                token = f"Bearer {cont["tokens"]["access"]}"
```